pandas/core/common.py

```diff
@@ -2455,6 +2455,12 @@ def is_list_like(arg):
     return (hasattr(arg, '__iter__') and
             not isinstance(arg, compat.string_and_binary_types))
 
+def is_dict_like(arg):
+    '''
+    Determin whether ``arg`` is dict-like.
+    '''
+    return hasattr(arg, '__getitem__') and hasattr(arg, 'keys')
+
```

Comment:
does this work for py3 dictviews? (maybe add a confirming tests in test_common.py)